### ISSUE TITLE
[Bug] minecraft crashes with xaeros Mini-map and or worldmap mods

### ISSUE THREAD

**Issue by puffycheezball8** [PojavLauncherTeam/PojavLauncher_iOS#481]
### Describe the bug

minecraft crashes when joining a world/server with xaeros Mini-map and or worldmap

### The log file and images/videos

Log file:
https://mega.nz/file/lMRzWRLL#u9eImuozMwmgE9n21Q-Bp2iO9ylvhPhhnLJ5C7cl9FA

### Steps To Reproduce

```markdown
1. make sure xaeros worldmap and or Mini-map is install
2. Load minecraft 1.19.2 fabric
3. Load world/server
4. See crash and log
```

### Expected Behavior

World should load

### Platform

```markdown
- Device model: iPhone 14 Pro
- iOS version: 16.2
- PojavLauncher version: 2.1.2
- Minecraft Version: 1.19.2, modded with fabric
```

**Comment by khanhduytran0** [comment]
Please upload logs directly to GitHub. Mega takes whole lot for time to load.

> aused by: xaero.common.exception.OpenGLException: OpenGL error: 1280

A conflict might have been there, or the renderer might not support some required functions

**Comment by puffycheezball8** [comment]
The log exceeded GitHub’s body length

**Comment by khanhduytran0** [comment]
Nah, you don’t upload log that way. Instead, tap 🖼️ and send txt file (not screenshot!)
